### PR TITLE
Redocly intro text

### PR DESCRIPTION
**Pull request by jhadobe**
This PR adds introductory text to the REST schemas page hosted by Redocly to the `2.4.7` schemas. The `2.4.6` schemas will be addressed in another PR.

I've also modified the .redocly.yaml file to include the `2.4.7-beta` files.

working `2.4.7`: https://preview.redoc.ly/adobe-commerce/jh_redocly_note/2.4.7-beta-admin/

### CODE REVIEW

**Comment by jhadobe** [comment]
@jeff-matthews there seems to be a difference in validation between `2.4.6` and `2.4.7-beta` versions. The betas all handle a reference to a description file correctly, while the `2.4.6` versions all error with a test failure of a file not found. 

The only way I have found to get them to work is by using an external link that links directly to an `.md` file in this repo (see `customer-schema-2.4.6.yaml` for example).

**Review by jeff-matthews — COMMENTED** [review]
That's bizarre 😆 

So we would need to first merge the new file for 2.4.7-beta, then come back and add the URL? Or could we go ahead and add an anticipated raw github URL now?

**Comment by jhadobe** [comment]
> That's bizarre 😆
> 
> So we would need to first merge the new file for 2.4.7-beta, then come back and add the URL? Or could we go ahead and add an anticipated raw github URL now?

Yeah, if we are ok just linking to the github url, then i would suggest we break this into two PRs. The next PR would link all the `2.4.6` schemas to the github `.md` file added in this PR.

**Comment by jeff-matthews** [comment]
That works for me!

File: src/_includes/redocly-intro.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This documentation provides descriptions of all the Adobe Commerce REST API endpoints. These three different REST APIs (Admin, Customer, and Guest) determine which endpoints are accessible. Select the Adobe Commerce version and corresponding API that you want to view the documentation for.
```
Comment (jeff-matthews) [inline src/_includes/redocly-intro.md:1]:
```suggestion
The schemas documented here are autogenerated from an instance of Adobe Commerce with B2B. Each schema represents a specific user role (Admin, Customer, and Guest) and determines which endpoints are accessible. Use the version switcher to select an Adobe Commerce version and corresponding API.
```

**Review by jeff-matthews — COMMENTED** [review]
I made some suggestions on the wording. What do you think?

**Comment by jhadobe** [comment]
> I made some suggestions on the wording. What do you think?

I like that wording much better!

**Review by jeff-matthews — APPROVED** [review]
🥳

**Comment by jeff-matthews** [comment]
What's even stranger is that when I run the Redocly validation locally on your commit with the 2.4.6 `$ref` object, i don't get any errors...

**Comment by jhadobe** [comment]
> What's even stranger is that when I run the Redocly validation locally on your commit with the 2.4.6 `$ref` object, i don't get any errors... Maybe it's a bug on the Redocly side.

yeah, i assume it is looking for the file in some temporary cache that does not have the same structure.

**Pull request merged**